lib/software_version/version.rb

```diff
@@ -84,7 +84,7 @@ def parse_raw_version(raw_version)
 
     # Parse the version to get the major, minor and patch parts
     def sv
-      @sv ||= version.scan(/(?:\d+|\D+)/)
```

Comment:
`\w` => `[a-zA-Z0-9_]` so `(?:\d+|\w+)` is equivalent to `(?:\w+)` and equivalent to `/\w+/`